pubspec.yaml

```diff
@@ -5,8 +5,8 @@ version: 0.3.0
 homepage: https://github.com/VeryGoodOpenSource/mockingjay
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=1.17.0"
+  sdk: ">=2.18.0 <3.0.0"
```

Comment:
```suggestion
  sdk: ">=2.19.0 <3.0.0"
```

Reply:
Is this necessary? People using this package will be forced to update flutter. Also, mocktail has a very permissive SDKconstraint.

Reply:
It’s not necessary since Very Good Analysis is a dev dependency but technically it requires dart 2.19 for the new lints. I’m fine merging without bumping the sdk version though 👍